velox/functions/prestosql/Sequence.cpp

```diff
@@ -35,6 +35,11 @@ int64_t toInt64(Date value) {
   return value.days();
 }
 
+template <>
+int64_t toInt64(Timestamp value) {
```

Comment:
const Timestamp& value,

Reply:
Unfortunately, If this is changed, the template parameter wouldn't match